examples/catalog/bin/sample_page.dart

```diff
@@ -105,27 +115,29 @@ class SampleInfo {
     final RegExp startRE = RegExp(r'^/\*\s+^Sample\s+Catalog', multiLine: true);
     final RegExp endRE = RegExp(r'^\*/', multiLine: true);
     final Match startMatch = startRE.firstMatch(contents);
-    if (startMatch == null)
-      return false;
+    if (startMatch == null) return false;
```

Comment:
Not an improvement

Reply:
I've added a follow up CL that automatically adds curly braces for all single line if statements.